types/react-input-mask-format/index.d.ts

```diff
@@ -1,7 +1,6 @@
 // Type definitions for react-input-mask-format 1.0
 // Project: https://github.com/temirtator/react-input-mask-format
 // Definitions by: Temirlan Shagyrov <https://github.com/temirtator>
-//                 Nikita Lobachev <https://github.com/sanniassin>
```

Comment:
Since I decided to maintain the source code myself, I think it would be correct to write only myself.